test/core/fixtures/transformation/es7.function-bind/call/expected.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+var _context;
+
+ns.obj.func.call(ctx);
+(_context = ns.obj).func.call(_context);
+
+ns.obj1.func.call(ns.obj2);
```

Comment:
> You'll also have to memoise single identifier references that aren't bound (generateMemoisedReference will be of use here) since referencing it may trigger a globally set getter.

@sebmck I just used same memoisation technique as for any other case - scoped identifier is rather specific case optimization that can be done later if needed. For now, it's more interesting to get correct behavior and feedback from users asap so that we could get this operator in ES7 or not.